### PR TITLE
Feature/main screen

### PR DESCRIPTION
**Pull request by Roller23**
Key changes so far:

- Changed the font from Lato to Inter
- Added the ability to sign in by clicking enter instead of clicking the button
- Added status messages
- Added input animations
- Added fade out animation
- You're logged in automatically after logging in once (you can log off later)

ezgif-7-981ea3da6854

image

This is a draft and probably should not be merged just yet

To do:

- implement a basic main hub

### CODE REVIEW

**Comment by adacienciala** [comment]
Overall looks great! Can the dots in "Authorizing..." be animated?

**Comment by Roller23** [comment]
@adacienciala yeah I'd like to see some kind of animation here, either these three dots changing or some spinner like

image

would be pretty cool

**Comment by Roller23** [comment]
OR maybe... mini rain animation

File: scripts/javascript/utils.js [hunk 1]
```diff
@@ -48,3 +52,17 @@ function authSocket() {
     });
 }
 exports.authSocket = authSocket;
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+exports.sleep = sleep;
+function fadeOut(el, ms) {
```
Comment (byteingWolf) [inline scripts/javascript/utils.js:71]: Hardcoded time handling could create a problem in future

**Review by byteingWolf — APPROVED** [review]
Overall looks great but please type comments where funtion are termopary and/or you are not sure about their usage in future.

File: scripts/javascript/utils.js (hunk 1 above)
Comment (Roller23) [reply scripts/javascript/utils.js:71]: Thanks for the heads up, fixed

**Comment by Roller23** [comment]
Progress report

- it's now possible to send messages to a general test channel
- some fixes

Screenshot 2021-04-07 at 22 15 29

- New scrollbar look proposed by me (tbc)

Screenshot 2021-04-07 at 22 15 58

Still no hub

**Comment by byteingWolf** [comment]
There is a problem with filling text boxes after inncorrect login attempt. 

https://user-images.githubusercontent.com/69678128/115070240-fab05100-9ef4-11eb-87e6-290588b3111e.mp4



Lets try correct it :)

**Comment by Roller23** [comment]
Progress update:

Implemented the new Figma layouts partially, added animations and tooltips


https://user-images.githubusercontent.com/25384028/120728161-95341600-c4dc-11eb-831b-a670a2685dfc.mov

**Comment by Roller23** [comment]
This PR is going for far too long, I think it's better to merge it and then split work into separate branches and PRs

**Review by szezjo — APPROVED** [review]
Yes.

**Pull request merged**